### ISSUE TITLE
Distribution - Clarify the criteria for downstream projects to be called distributions

### ISSUE THREAD

**Issue by jpkrohling** [open-telemetry/opentelemetry-collector#10640]

**Comment by jpkrohling** [comment]
Is this a real problem? At this stage, it would be helpful for the community (contributors, users) to help us understand how this affects them. If you believe this is a problem, or has been affected by this, vote this comment with 👍🏽 . If you think this is NOT a problem, vote this comment with 👎🏽

**Comment by codefromthecrypt** [comment]
I might suggest re-wording this part of the description. I am not sure if the key part here is vendors want to be labeled as "supported" rather want to use a word like "distribution" so that it is non-confusing what this thing is.

> The vendors don't want to support a binary they don't ship and want to call their products "Distributions" so that people have confidence they are using a "supported Collector."

For example, "X.com distribution of opentelemetry collector" or "X.com opentelemetry collector distribution", not so much that users feel that the upstream "support it" rather that they know what they are likely getting (a collector distribution). I think that one vendor not supporting another's distribution of something is common enough to be a background, not a foreground concern. Also, if the word distribution exists near the company name, I'm not sure there is an implicit expectation of the opentelemetry main channels to "support" it.

TL;DR; I would like maybe to clarify if this is about branding of the word distribution (already used so hard to police even if we wanted to), or some shared support connotation if someone uses that word (I would downvote that for sure).

**Comment by codefromthecrypt** [comment]
As mentioned on the other issue, I think that if we are looking for some sort of vouching for a distribution, it would be via some certification (likely certified semantics 1.27 or whatever test suite one comes up with).

Meanwhile, focus this issue on the word distribution which by itself has not a very good way to "certify", so maybe should just be left to the world of brand abuse vs qualitative things like it is supported or not. That's somewhat easy to navigate and common issue since before CNCF was a thing.

My 2p on routes forward.
